lms/djangoapps/courseware/toggles.py

```diff
@@ -129,22 +132,79 @@
 )
 
 
+def courseware_mfe_is_active(course_key: CourseKey) -> bool:
+    """
+    Should we serve the Learning MFE as the canonical courseware experience?
+    """
+    # NO: Old Mongo courses are always served in the Legacy frontend,
+    #     regardless of configuration.
+    if course_key.deprecated:
+        return False
+    # OTHERWISE: Defer to value of waffle flag for this course run and user.
+    return REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key)
```

Comment:
Other than tests, the `courseware_mfe` Waffle flags is only checked in toggles.py now. This will make it easier to invert the toggle into `use_legacy_frontend` in an upcoming PR.